packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -72,7 +110,15 @@ export class PlaygroundV2Manager extends Publisher<PlaygroundV2[]> implements Di
     } else if (server.status === 'stopped') {
       await this.inferenceManager.startInferenceServer(server.container.containerId);
     }
+
```

Comment:
You should propagate the trackingId you created to the InferenceServer to see all the tasks related to the inference server creation.

https://github.com/projectatomic/ai-studio/blob/faf50e4bc7359728195b99cd8080e608e47bb52c/packages/backend/src/managers/playgroundV2Manager.ts#L72-L77

Reply:
ah yes, thanks, I was looking at how to do this